deploy/base/run-maven-component-build-v0.1.yaml

```diff
@@ -1,266 +0,0 @@
-apiVersion: tekton.dev/v1beta1
-kind: ClusterTask
```

Comment:
We can't merge with this deletion until https://github.com/redhat-appstudio/infra-deployments/pull/525 is also merged.

Reply:
I would have guessed that it would have been part of whatever pull request inlined the yaml. I am not sure how it ended up here.

Reply:
Sorry @dwalluck from my point of view how it ended up here is a bit irrelevant at this point.

For me at least, per my earlier comment around the number of commits you now have, I think at this point it is time to 
- squash your now 32 commits locally via `git rebase main -i` 
- `git reset HEAD`
-  'git checkout deploy/base/run-gradle-component-build-v0.1.yaml`
- `git checkout deploy/base/run-maven-component-build-v0.1.yaml`
- `git commit -a -m "rebuild java dependencies that use gradle"
- `git push -f origin PLNSRVCE-346`

And then we will handle these deletes via a PR and commit message that conveys they are being deleted.

But if I'm in the minority on this opinion @dwalluck @stuartwdouglas, based on the reasoning why,  I could be convinced to defer to you two.

Reply:
and just remembered / rediscoverd - @stuartwdouglas already has a yaml cleanup PR that is taking care of this file:  https://github.com/redhat-appstudio/jvm-build-service/pull/136

it got hit by some of the bugs that between he and I we diagnosed / fixed

I will see if I can rebase that, getting it passing e2e's and merged